Jenkinsfile

```diff
@@ -1089,7 +1089,7 @@ pipeline {
                         }
                     }
                 }
-                stage('Build on Leap 15 with Intel-C and TARGET_PREFIX') {
+                stage('Build on Leap 15 with Intel-C') {
```

Comment:
this requires a change in GitHub to the required stages

Reply:
Reverted.  If we're forcing people to rebase we should make this change at the same time.